content/resources/faq.md

```diff
@@ -28,6 +35,12 @@ GitBOM seeks to have the following characteristics:
 
 The uniqueness requirement is what drives the exclusion of metadata from the GitBOM.
 
-Metadata out of tree can always reference artifacts by their artifact ID or parent child relationships by the GitBOM id.
+Metadata, which is stored outside of the GitBOM tree, may reference any artifact within the tree by its artifact ID, and can also reference a parent-child relationship in the same way.
+
+## Won't the generation of artifact trees slow down build processes?
+
+We don't think so, and would be delighted to receive data from any very large projects which would either challenge or validate this assumption.
+
+## What about files with duplicate hashes?
 
-### Why isn't information about the compiler/linker included in the GitBOM?
\ No newline at end of file
+We don't think this will be a problem because GitBOM is not an SBOM -- it does not provide provenance metadata about any files or artifacts, which is the domain of SBOM tools and standards such as SPDX and CycloneDX. While duplicate hashes of empty files and files which are regularly copied (such as LICENSE files), this does not affect the security properties of GitBOM.
```

Comment:
Are we concerned here about non-equivalent files with duplicate hashes?  If so I'd suggest we point to GitHUBs article that points out they've never seen duplicate hashes in the wild for non-equivalent files.

Reply:
No -- I've had two folks ask me about duplicate hashes for equivalent files, so added this to address that question.

If one were looking at gitbom as though it were a form of file identification, then it would be reasonable to be concerned that two files with different provenance, and possibly different licenses, could have the same hash. This is also why I removed the note about license discovery.

From a file identification perspective, when looking at using gitbom for very large scale scanning, duplicate files across projects will happen -- just imagine how many instances of the apache2 license.txt file will have the same hash. This is going to happen, but it won't be a _problem_ for the reasons I tried to explain.

Reply:
Got it.